### PR TITLE
Fix(pisa-daemon): fix incorrect classid in tc command

### PR DESCRIPTION
**Pull request by xuanyuan300**
Signed-off-by: xuanyuan300 <xuanyuan300@gmail.com>

### What is changed and how it works?

What's Changed:
1. Fix classid in tc command.
2. Add config.toml example.
3. Changed  APP_ENDPOINTS_CLASSID_PIN_PATH to avoid iproute do not read bpf map.

### CODE REVIEW

File: pisa-daemon/app/traffic_qos/src/qos.rs [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                obj: "app.o",
+        });
+        add_filter(&FilterAttr {
+                netns: None,
+                device: self.config.global.egress_device.as_str(),
+                obj: "app.o",
+        });
+
+        // load
+        println!("endpoint classid {:?}", endpoint_classid);
```
Comment (mlycore) [inline pisa-daemon/app/traffic_qos/src/qos.rs:151]: using Crate tracing

File: pisa-daemon/bpf/src/load.rs [hunk 2]
```diff
@@ -63,7 +62,7 @@ pub enum TrafficTyp {
     SQL,
 }
 
-pub const APP_ENDPOINTS_CLASSID_PIN_PATH: &str = "/sys/fs/bpf/pisa-daemon/app-endpoints-classid";
+pub const APP_ENDPOINTS_CLASSID_PIN_PATH: &str = "/sys/fs/bpf/tc/globals";
```
Comment (mlycore) [inline pisa-daemon/bpf/src/load.rs:65]: Is this should be corrected to `/sys/fs/bpf/pisa-daemon/app-endpoints-classid` ?

File: pisa-daemon/bpf/src/load.rs [hunk 3]
```diff
@@ -134,13 +134,13 @@ mod test {
         let _ = Command::new("clang").args("-O2 -target bpf -g -c tc/app.c -o tc/app.o -I ./".split(" ")).spawn();
         let load = TrafficTyp::App;
         load.add_clsact(&vec!["lo"]).unwrap();
-        let try_bpf = load.app("tc/app.o", &vec!["lo"]);
+        let try_bpf = load.app("tc/app.o", &vec!["cni0"]);
```
Comment (mlycore) [inline pisa-daemon/bpf/src/load.rs:137]: Is this network interface should be loaded from the global config ?

**Review by mlycore — APPROVED** [review]

File: pisa-daemon/bpf/src/load.rs (hunk 2 above)
Comment (xuanyuan300) [reply pisa-daemon/bpf/src/load.rs:65]: This is `tc` default value when pinning is `PIN_GLOBAL_NS` or `2`.

**Pull request merged**